controllers/controller/workspacerouting/workspacerouting_controller.go

```diff
@@ -35,15 +35,24 @@ import (
 	controllerv1alpha1 "github.com/devfile/devworkspace-operator/apis/controller/v1alpha1"
 )
 
-var externalRoutingError = errors.New("routingclass not supported by this controller")
+var ExternalRoutingError = errors.New("routingclass not supported by this controller")
```

Comment:
From the code, it would be clearer if it's named `RoutingNotSupported` or something like that. Then the doc says that controllers should throw it when they do not support the configured routing, so, it's expected to be handled by other controllers.